domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevbestiller/impl/DokgenLanseringTjeneste.java

```diff
@@ -11,11 +11,12 @@
     private static final Environment ENV = Environment.current();
     private static final Set<DokumentMalType> DOKGEN_MALER_PROD = Set.of(
             DokumentMalType.INNVILGELSE_ENGANGSSTØNAD,
-            DokumentMalType.IKKE_SØKT);
+            DokumentMalType.IKKE_SØKT,
+            DokumentMalType.AVSLAG_ENGANGSSTØNAD);
```

Comment:
Hvis du sørger for å ha samme rekkefølge her som du har endret til under, så blir det lettere å se kjapt hvilke som er lanserte og ikke